packaging/post_build_script.sh

```diff
@@ -7,15 +7,14 @@
 
 set -eux
 
-WHEEL_NAME=$(ls dist/)
+# Prepare manywheel, only for CUDA.
+# The wheel is a pure python wheel for other platforms.
+if [[ "$CU_VERSION" == cu* ]]; then
```

Comment:
is this also true for rocm?

Reply:
No, but it looks like the current .cu files are not built for rocm based on the `use_cuda` here https://github.com/pytorch/ao/blob/main/setup.py#L64

Reply:
Ok! heads up to @petrex who has been working on adding custom hip kernel support

Reply:
@janeyx99 This https://github.com/pytorch/ao/pull/1201 would build hip kernels.